packages/terra-overlay/translations/de.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I googled them. Where do I find translations/who would I reach out to on the translations team?